UICatalog/Scenarios/InteractiveTree.cs

```diff
@@ -10,52 +10,47 @@ public class InteractiveTree : Scenario
 {
     private TreeView _treeView;
 
-    public override void Setup ()
+    public override void Main ()
     {
-        Win.Title = GetName ();
-        Win.Y = 1; // menu
-        Win.Height = Dim.Fill (1); // status bar
+        var appWindow = new Toplevel ()
```

Comment:
Please add `Application.Init ();` into the scenario to avoid exception been throwing.